webrtc.html

```diff
@@ -11641,6 +11642,13 @@ <h2 id="sec.idp-error-handling">IdP Error Handling</h2>
                 <td>The data channel has failed.</td>
               </tr>
               <tr>
+                <td><dfn><code>dtls-failure</code></dfn></td>
+                <td>The DTLS negotiation has failed or the connection
+                has been terminated with a fatal error. The 
+                <code>tlsAlert</code> attribute is set to the
+                DTLS alert value.</td>
+              </tr>
```

Comment:
As discussed on the call:

1. I think we should have a "sentAlert" and "receivedAlert" value because both can happen.
2. Note that close_notify is alert code 0. Maybe "none"?